src/main/java/com/xero/api/client/AppStoreApi.java

```diff
@@ -2,7 +2,7 @@
  * Xero AppStore API
  * These endpoints are for Xero Partners to interact with the App Store Billing platform
  *
- * The version of the OpenAPI document: 2.17.2
+ * The version of the OpenAPI document: 2.19.1
```

Comment:
missed one of the spec files when updating version?

Reply:
Missed it when updating the OpenAPI spec. Resolved now